<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aFoods</class>
 <widget class="QWidget" name="aFoods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="aFoodTV">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>701</width>
     <height>531</height>
    </rect>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="foodDBLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Segoe UI Emoji&quot;;</string>
   </property>
   <property name="text">
    <string>Food Database</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>381</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="addLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>15</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Franklin Gothic Demi&quot;;
</string>
      </property>
      <property name="text">
       <string>Add Food Item</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="addFields">
      <item>
       <layout class="QHBoxLayout" name="citySection">
        <item>
         <widget class="QLabel" name="cityLabel">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>City:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cityCB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="nameSection">
        <item>
         <widget class="QLabel" name="foodLabel">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Food Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="foodLE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="costSection">
        <item>
         <widget class="QLabel" name="costLE">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Cost:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="costSB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <double>9999999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonSection">
        <item>
         <widget class="QPushButton" name="cancelAdd">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addFood">
          <property name="text">
           <string>Add Food</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addFile">
          <property name="text">
           <string>Add From File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="delUpLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Franklin Gothic Demi&quot;;
</string>
      </property>
      <property name="text">
       <string>Delete and Update Food Item</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="deleteFields">
      <item>
       <layout class="QVBoxLayout" name="deleteChoices">
        <item>
         <layout class="QHBoxLayout" name="delCityField">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="delCityLabel">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>City: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="delCityCB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="foodLV"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="delCost">
        <item>
         <widget class="QLabel" name="delCostLabel">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Price:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="delCostSB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <double>9999999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="deleteButtons">
        <item>
         <widget class="QPushButton" name="cancelDel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delButton">
          <property name="text">
           <string>Delete Food</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateButton">
          <property name="text">
           <string>Update Price</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
